src/main/java/me/ford/iwarp/WarpHandler.java

```diff
@@ -215,4 +218,36 @@ public OfflinePlayer getOwner(String name) { // I'm allowing owners not managed
 		return IW.getServer().getOfflinePlayer(owner);
 	}
 
+	public boolean isProhibitedName(CommandSender sender, String warpName) {
+		if (warpName.contains(".")) {
+			sender.sendMessage(IW.getSettings().getNameContainsPeriodMessage(warpName));
+			return true;
+		}
+
+		try {
+			if (!warpName.matches(IW.getSettings().getWarpNameFormat())) {
+				sender.sendMessage(IW.getSettings().getNameDoesntMatchPatternMessage(warpName));
+				return true;
+			}
+		} catch (PatternSyntaxException exception) {
+			Bukkit.getLogger().warning("You have an error in the warp-name-format configuration setting. Until this error is fixed, iwarp will allow any warp name. You can reset it to \"^.{1,15}$\".");
```

Comment:
There should probably be a return statement after here. Probably `false` by default.
Otherwise it'll depend on the subsequent code, which may not be desirable.